demo/index.html

```diff
@@ -25,6 +27,52 @@
     <link rel="stylesheet" type="text/css" href="css/style.css">
   </head>
   <body>
+      <div class="vertical-section-container">
+          <h1>auro-button</h1>
+    
+          <!-- examples for shadow DOM element -->
+    
+          <h3>Shadow DOM element examples</h3>
+          <demo-snippet>
+            <template>
+              <auro-button>Default state</auro-button>
+              <auro-button secondary>Secondary Default state</auro-button>
+              <auro-button tertiary>Tertiary default state</auro-button>
+            </template>
+          </demo-snippet>
+    
+          <demo-snippet class="dark">
```

Comment:
This selector is not doing anything? 

Also, should be named `onDark` to be consistent.

Reply:
Removed the dark class. As discussed, @blackfalcon will be looking at this as we cannot style demo-snippet in its current state.